### ISSUE TITLE
release v1.13.1 lacks release status

### ISSUE THREAD

**Issue by rofl0r** [hrydgard/ppsspp#15761]
### Platform

Linux / BSD

### Compiler and build tool versions

irrelevant

### Operating system version

irrelevant

### Build commands used

irrelevant

### What happens

a github CI action was added to create a proper release tarball including all sources; however it requires that one clicks the button "make release" in github and selects the according tag. this was done for 1.13, but not for 1.13.1, therefore the release source tarball is missing.

so, please click that button.

### PPSSPP version affected

v1.13.1

### Last working version

v1.13

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] Make sure to run `git submodule update --init --recursive` before building.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try deleting the build directory and running the build again.
- [X] Check GitHub Actions, which builds every merge and PR.
- [X] Include logs and help us reproduce.

**Comment by hrydgard** [comment]
But I can't click Make Release anymore, since the release was already auto-created.

If I do what you say, like I did for 1.13, there will instead be no MacOS release.

So something is quite screwed up.... See #15737, maybe you have some useful input.

**Comment by rofl0r** [comment]
it appears the culprit is https://github.com/hrydgard/ppsspp/blob/a92e764c65b3248d1ddd7107ffa4c598a2ef4ff2/.github/workflows/build.yml#L295 - the action to upload mac release tarball triggers a "automatically make release from tag" action which in turn doesn't trigger the manual release thing. i'll look into whether triggering the tarball action also on tag push fixes things.

**Comment by hrydgard** [comment]
happy to accept fixes in the form of PRs, if you figure out how it should be set up :)

**Comment by rofl0r** [comment]
now we only need a release tarball that contains the 1.13.1 bugfixes... would you be open to cherry-pick this commit to a 1.13.1 based branch and then tag it v1.13.2 ? because last time it took half a year for a new release to ship.

**Comment by hrydgard** [comment]
I suppose I could just delete the release and tag, and tag the same commit again?

**Comment by rofl0r** [comment]
> I suppose I could just delete the release and tag, and tag the same commit again?

no, that wouldn't work because the CI actions are taken from the tree at the time of the commit, not from master.

also deleting a tag should never be done, as this would change the checksum of the downloads.
precisely this happened when you deleted the 1.13 tag 3-4 days ago in order to get the mac binary back.
which in turn breaks distro packages [such as mine](https://github.com/sabotage-linux/sabotage/commit/4758cc87b644acec74618704e81c94026c7d588f) .
i'll have to update that package now to change the hash of the tar.xz (tar contains timestamps, thus its different).

optimally you'd make a branch named like `maintainance_1.13` based on 1.13.1, add the tarball.yml changes i did here, and tag that commit v1.13.1b or similar.

**Comment by rofl0r** [comment]
the instructions to do so are:

```
git checkout v1.13.1
git checkout -b maintainance_1.13
git cherry-pick 8626780
git tag v1.13.1b
git push origin maintainance_1.13 v1.13.1b
```

**Comment by hrydgard** [comment]
Ah I see, I thought it wouldn't be a big deal to re-tag because it is the same commit after all, but I guess other things like timestamp change so some kind of hash still changes.

Will do that.

**Comment by unknownbrackets** [comment]
Or one could just manually run the steps from the action locally, and upload the tar manually this time.

-[Unknown]

**Comment by hrydgard** [comment]
I'm too lazy (or rather, busy with other things) for that, I ran the suggested commands, hope it works.

**Comment by unknownbrackets** [comment]
I'm a bit worried that an actual tag 1.13.1b will cause all the versions henceforth to show as i.e. v1.13.1b-700-gabcdef.  I'll just move the tar from the new tag and then delete it.

-[Unknown]

**Comment by hrydgard** [comment]
That shouldn't happen because it's not on the master branch.

**Comment by unknownbrackets** [comment]
Ah, sorry, it was a cherry-pick.  Well, I deleted it anyway.

-[Unknown]

**Comment by rofl0r** [comment]
thanks to both of you, the tarball works perfectly 👍
issue is finally resolved and source users can wait for 1.14 with peace of mind.
